torch_geometric/loader/utils.py

```diff
@@ -56,7 +56,11 @@ def to_csc(
         colptr, row, _ = data.adj_t.csr()
 
     elif hasattr(data, 'edge_index'):
-        (row, col) = data.edge_index
+        if data.edge_index is None:
+            edge_index = torch.empty((2, 0), dtype=torch.long)
```

Comment:
In this case, can't we just return `row, col, None`?

Reply:
@mananshah99 I added support for GraphStore without edges. it simply checks if `get_all_edge_attrs` returns a non-empty list. WDYT?

Reply:
Do you mean set `row` and `col` to None and return that?

Reply:
No, I just meant that we avoid trying to sort here

Reply:
Good point. I went ahead and skipped the `to_csc` if the graph doesn't have edges. I feel like this is more readable than setting empty tensors in `to_csc`